aptos-move/framework/aptos-framework/sources/aptos_account.move

```diff
@@ -132,6 +133,40 @@ module aptos_framework::aptos_account {
         coin::deposit<CoinType>(to, coins)
     }
 
+    /// Batch version of transfer_fungible_assets.
+    public entry fun batch_transfer_fungible_assets(
+        from: &signer,
+        metadata: Object<Metadata>,
```

Comment:
It this the convention that `metadata` is put right after signers and before anything else?

Reply:
Greg just wanna make the receiver and amount together